### PR TITLE
{devel,lib}[GCCcore/8.2.0,foss/2019a] PyTorch v1.1.0, psutil v5.6.3 w/ Python 2.7.15 + 3.7.2

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)
mostly copied from #8321

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic402 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/d64a1fa4ad4db05f49ebd5e2048f522f for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 15 out of 15 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/a13e9fcb3a5266b31f308d0a0ee519b7 for a full test report.

File: easybuild/easyconfigs/p/PyTorch/PyTorch-1.1.0-foss-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+preinstallopts = prebuildopts
+
+runtest = 'export PYTHONPATH=%(builddir)s/%(namelower)s-%(version)s/build/lib.linux-x86_64-%(pyshortver)s:$PYTHONPATH '
+runtest += '&& export LD_LIBRARY_PATH=%(builddir)s/%(namelower)s-%(version)s/torch/lib64:$LD_LIBRARY_PATH '
+runtest += '&& python test/run_test.py --verbose'
+
+options = {'modulename': 'torch'}
+
+postinstallcmds = ['cp -a %(builddir)s/%(namelower)s-%(version)s/torch/lib64/* %(installdir)s/lib/']
+
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PyTorch/PyTorch-1.1.0-foss-2019a-Python-3.7.2.eb]: @smoors Don't we need a sanity check here?

Comment (smoors) [reply easybuild/easyconfigs/p/PyTorch/PyTorch-1.1.0-foss-2019a-Python-3.7.2.eb]: @verdurin good idea, I'll add it.

**Comment by verdurin** [comment]
Going in, thanks @smoors!

**Pull request merged**